sdk/agrifood/arm-agrifood/vitest.config.ts

```diff
@@ -0,0 +1,15 @@
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { defineConfig, mergeConfig } from "vitest/config";
+import viteConfig from "../../../vitest.shared.config.ts";
+
+export default mergeConfig(
+  viteConfig,
+  defineConfig({
+    test: {
+      include: ["test/**/*.spec.ts"],
+    },
+  }),
+);
```

Comment:
This can be simplified as follows:
```suggestion
export default viteConfig;
```

Reply:
Better default now with timeouts.